Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.h

```diff
@@ -610,6 +611,10 @@ class MediaPlayerPrivateGStreamer : public MediaPlayerPrivateInterface
     AbortableTaskQueue m_sinkTaskQueue;
 
     bool m_didTryToRecoverPlayingState { false };
+
+    // Specific to MediaStream playback.
+    MediaTime m_startTime;
+    MediaTime m_pausedTime;
```

Comment:
Shouldn't we flag this for `MEDIA_TRACK`?

Reply:
`MEDIA_STREAM` you mean? Yes, it wouldn't hurt, I guess :)